client/lib/signup/private-by-default.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:signup:private-by-default' );
+
+interface PrivateByDefaultSiteSettings {
+	flowName?: string;
+}
+
+/**
+ * Should the site be private by default
+ * @param siteSettings PrivateByDefaultSiteSettings
+ * @returns `true` for private by default & `false` for not
+ */
+export function shouldBePrivateByDefault( {
+	flowName = '',
+}: Readonly<PrivateByDefaultSiteSettings> ): boolean {
+	if ( flowName.match( /^ecommerce/ ) ) {
```

Comment:
This isn't a blocker in any way, but I was just thinking about this line because I'm using a variant of the `ecommerce` flow in my [user step A/B test](https://github.com/Automattic/wp-calypso/pull/34770/files#diff-28b03eff6ef567c591024d6033cda11d) where I've named the flow `online-store-onboarding`. So to keep PBD switched off, I'll rename that flow.

But, it made me think, would it be better to match against the `site_segment` instead of the flowName here, to determine whether or not to use private by default, just in case we have any other flow naming issues in the future? It probably isn't a big deal anyway, because it's easy for us to keep track of a handful of flows, so feel free to disregard this one! I've made a note to myself to update the flow name I'm using in my PR.

Reply:
You could be right about this, but I left it the way it was just because `site_segment` wasn't conveniently available as part of the step data object.